### PR TITLE
feat: Replace shortid with nanoid

### PR DESCRIPTION
**Pull request by AkshatJawne**
Resolves https://github.com/deephaven/deephaven-plugins/issues/568

**Changes Implemented:**
- Replace `shortid` package with `nanoid`

### CODE REVIEW

**Comment by AkshatJawne** [comment]
Looking into e2e

**Review by mattrunyon — CHANGES_REQUESTED** [review]
Looks like you have a package-lock conflict. Will need to merge main and then run `npm install --package-lock-only`

**Comment by AkshatJawne** [comment]
Ready for re-review, resolved package.lock errors

File: plugins/dashboard-object-viewer/src/js/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "@deephaven/jsapi-types": "^0.40.0",
     "@deephaven/log": "^0.40.0",
     "react-json-view": "^1.21.3",
-    "shortid": "^2.2.16"
+    "nanoid": "^5.0.7"
```
Comment (mattrunyon) [inline plugins/dashboard-object-viewer/src/js/package.json:27]: This one is still out of alphabetical order

**Review by mattrunyon — CHANGES_REQUESTED** [review]

File: plugins/dashboard-object-viewer/src/js/package.json (hunk 1 above)
Comment (AkshatJawne) [reply plugins/dashboard-object-viewer/src/js/package.json:27]: Hmm that's weird that it did not fix when I reinstalled, will fix now

**Review by mattrunyon — APPROVED** [review]

**Pull request merged**